Thirty/Thirty/View Controllers/Call/CallViewController.swift

```diff
@@ -49,11 +60,11 @@ class CallViewController: UIViewController, TVIRoomDelegate, TVIRemoteParticipan
         remoteVideoView.delegate = self
         CallManager.shared.delegate = self
         FirebaseManager.shared.delegate = self
+        setUpUI()
```

Comment:
Good call, because if for some reason we called viewWillAppear agin, it would bring some views back to their default alpha 0 settings.